app/src/main/java/com/example/dinhh/soundscape/device/Player.kt

```diff
@@ -12,12 +12,18 @@ interface Player {
     fun stopSound(): Completable
 
     fun onPlayComplete(): Completable
+
+    fun loopSound()
+
+    fun stopLoop()
```

Comment:
Just combine it to one function called fun setPlayerLooping(isLooping: bool): Completable

Reply:
You're right. I just fixed this one, but I'm just wondering is there any reason it should be completable? It just toggles one boolean on and off.